test/unit/hibf/interleaved_bloom_filter_test.cpp

```diff
@@ -16,28 +16,9 @@
 #include <hibf/interleaved_bloom_filter.hpp> // for bin_index, interleaved_bloom_filter, bin_count, bin_size, hash_...
 #include <hibf/test/expect_range_eq.hpp>     // for expect_range_eq, EXPECT_RANGE_EQ
 
-#include <sdsl/io.hpp> // for size_in_mega_bytes
+using TypeParam = seqan::hibf::interleaved_bloom_filter;
```

Comment:
I wouldn't call it TypeParam if it is not a type parameter but just an alias? 

```suggestion
using ibf_t = seqan::hibf::interleaved_bloom_filter;
```

Actually, I would even prefer to just use the type itself instead of an alias.